ctapipe/utils/tests/test_astro.py

```diff
@@ -0,0 +1,12 @@
+from ..astro import get_bright_stars
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+
+def test_get_bright_stars():
+
+    pointing = SkyCoord(ra=83.275 * u.deg, dec=21.791 * u.deg, frame='icrs')
+
+    table = get_bright_stars(pointing, radius=5. * u.deg, magnitude_cut=3.5)
+
+    assert len(table) == 1
+    assert table[0]['Name'] == '123Zet Tau'
```

Comment:
Just a question: is the cut of 5 degrees too large for this test? I assume you want to try to *only* get Zeta Tau.  You are not necessarily guaranteed to get the table back in the same order (even if that is the current way it's implemented, this might change, and shouldn't be a requirement), unless you say somewhere that the returns results are sorted by distance to the pointing?  Otherwise, i'd suggest a smaller radius so you are guaranteed to get only 1 result, so then table[0] must be Zeta Tau.

Reply:
Zeta Tauri should be the only star brighter than 3.5 in a 5 degree radius around crab. Also see the `assert len(table) == 1`

Reply:
ah, i missed the assert. It's fine then